src/widget/geo/geopicker.js

```diff
@@ -773,7 +773,6 @@ class Geopicker extends Widget {
         // update last requested map coordinates to be used to initialize map in mobile fullscreen view
         if (latLng) {
             this.lastLatLng = latLng;
-            this.lastZoom = zoom;
```

Comment:
I'm not _certain_ this isn't guarded here for a reason, but it seems like the event handler more likely is the expected behavior.